docker-compose.yml

```
@@ -1,27 +1,46 @@
+# Docker-compose configuration for development
+
 version: '3.8'
 
+x-common-spec: &common-spec
+  env_file: .env
+
+x-python-container-spec: &python-container-spec
+  <<: *common-spec
+  build:
+    context: .
+    dockerfile: Dockerfile
+    target: dev
+  environment:
+    PYTHONUNBUFFERED: "1"
+  volumes:
+    - ".:/app"
+  depends_on:
+    - postgres
+
+
 services:
-  web:
-    env_file: .env
-    build: ./app/
-    working_dir: /app/src/
-    command: python main.py
-    volumes:
-      - ./app/src:/app/src
+  python:
+    <<: *python-container-spec
     ports:
-      - '8000:8000'
-    depends_on:
-      - postgres
+      - "${FASTAPI_PORT-8000}:8000"
+    command: python3 src/main.py
```

After changing the mount point, I see a problem running this command. (It looks for `/app/src/main.py` in the container, while its real path is `/app//app/src/main.py`)
I believe we can just remove the intermediate `src` directory and put code just under `app` so that this line looks like 
```suggestion
    command: python3 app/main.py
```

---

I was inattentive. 
Fixed it.